sql/core/src/test/scala/org/apache/spark/sql/IntegratedUDFTestUtils.scala

```diff
@@ -621,10 +623,30 @@ object IntegratedUDFTestUtils extends SQLHelper {
         |""".stripMargin
   }
 
+  object UDTFPartitionByOrderBy
+    extends TestPythonUDTFPartitionByOrderByBase(
+      partitionBy = "partition_col",
+      orderBy = "input")
+
+  object UDTFPartitionByOrderByComplexExpr
+    extends TestPythonUDTFPartitionByOrderByBase(
+      partitionBy = "partition_col + 1",
+      orderBy = "RANDOM(42)")
+
+  object UDTFInvalidPartitionByOrderByParseError
+    extends TestPythonUDTFPartitionByOrderByBase(
+      partitionBy = "unparsable",
+      orderBy = "input")
+
+  object UDTFInvalidOrderByAscKeyword
+    extends TestPythonUDTFPartitionByOrderByBase(
+      partitionBy = "partition_col",
+      orderBy = "partition_col ASC")
```

Comment:
Where are these new tests called?

Reply:
Apologies about that, it seems it was a bad merge from the last PR. I fixed this now, and the tests are used.